unix-domain-socket/src/main/scala/akka/stream/alpakka/unixdomainsocket/impl/UnixDomainSocketImpl.scala

```diff
@@ -406,15 +412,17 @@ private[unixdomainsocket] abstract class UnixDomainSocketImpl(system: ExtendedAc
 
       val serverBinding = Promise[ServerBinding]
 
-      val channel = UnixServerSocketChannel.open()
+      val channel = UnixSocketSelectorProvider.openUnixServerSocketChannel()
       channel.configureBlocking(false)
-      val address = new UnixSocketAddress(file)
+      val address = UnixSocketAddress(path)
       val registeredKey =
         channel.register(sel,
                          SelectionKey.OP_ACCEPT,
                          acceptKey(address, incomingConnectionQueue, halfClose, receiveBufferSize, sendBufferSize) _)
       try {
-        channel.socket().bind(address, backlog)
+        val socket = channel.socket()
+        socket.bind(address)
+        socket.listen(backlog)
```

Comment:
JNR conflated `bind` and `listen`. We now expand it out as per conventional usage.